workflow/5_adaptation/adaptation_options_costs.py

```diff
@@ -0,0 +1,381 @@
+"""Estimate adaptation options costs and benefits
+
+"""
+
+import click
+import os
+import logging
+import warnings
+
+import pandas as pd
+
+import geopandas as gpd
+import numpy as np
+from tqdm import tqdm
+
+from jamaica_infrastructure.analysis.utils import (
+    calculate_discounting_rate_factor,
+)
+
+warnings.simplefilter(action="ignore", category=FutureWarning)
+pd.options.mode.chained_assignment = None  # default='warn'
+warnings.simplefilter(action="ignore", category=pd.errors.PerformanceWarning)
+
+tqdm.pandas()
+
+
+def assign_maintenance_cost_over_time(
+    df,
+    maintenance_intervals_years,
+    maintenance_cost,
+    start_year=2019,
+    end_year=2100
+):
+    maintain_intervals = list(
+        set(df[maintenance_intervals_years].values.tolist())
+    )
+
+    if len(maintain_intervals) > 1:
+        df_maintain = []
+        for interval in routine_maintain_intervals:
```

Comment:
`routine_maintain_intervals` isn't defined anywhere in this file, and isn't imported either.

It doesn't crash the script, which makes me think we can safely delete this block.